addons/web_editor/static/src/js/backend/convert_inline.js

```diff
@@ -792,6 +792,10 @@ function flattenBackgroundImages(editable) {
             // Hide the original element for Outlook.
             backgroundImage.classList.add('mso-hide');
         }
+        if (backgroundImage.hasAttribute('data-bg-src')) {
+            // Fix Gmail cuts html when it's too long because of data-bg-src
```

Comment:
This reads like a commit title instead of a comment... ^^' I suggest (wrap at 80 chars):
```
// Remove data-bg-src as it is not needed for email rendering and can cause Gmail to cut the email prematurely if the attributes contain an image in the form of a long base64 string.
```